doc/release/1.17.0-notes.rst

```diff
@@ -90,6 +90,38 @@ C API changes
 New Features
 ============
 
+libFLAME
+--------
+Enabled the use of the libFLAME linear algebra package as a drop-in
+replacement for NetLIB LAPACK, see
```

Comment:
Perhaps: "Support for building NumPy with the libFLAME linear algebra package as the LAPACK implementation" ?